dragon-scales/src/app/components/TaskViewerComponents/TaskViewer.jsx

```diff
@@ -1,223 +1,208 @@
-                                    }}
-                            >
-                                Submit
-                            </Button>
-                        </StyledNewContentBox>
-                    </form>
-                    <Box display="flex" flexDirection="column" alignItems="center">
-                        <Divider sx={{ width: "90%", margin: "auto", }} />
-                        <IconButton aria-label="add new entity"
-                                    sx={{ paddingTop: 1 }}
-                                    onClick={handleOpenNewEntityDialog}>
-                            <AddBoxOutlinedIcon titleAccess="Add new entity" />
-                        </IconButton>
-                    </Box>
+                <Stack direction="row" alignItems="center" justifyContent="space-between" paddingX={2}>
+                    <StyledTaskTitleTypography>{task.name}</StyledTaskTitleTypography>
+                    <StyledButtonStack direction="row" spacing={1}>
+                        <StyledEditButton onClick={handleOpenEditDialog} aria-label="edit task">
+                            <EditIcon />
+                        </StyledEditButton>
+                        <StyledDeleteButton onClick={handleOpenDeleteDialog} aria-label="delete task">
+                            <DeleteIcon />
+                        </StyledDeleteButton>
+                    </StyledButtonStack>
+                </Stack>
+                <Breadcrumbs separator=">" color="#4C9DFC" paddingLeft={2} paddingTop={1}>
+                    {breadcrumbsText.map((text) => (
+                        <Typography key={text} color="#000000">
+                            {text}
+                        </Typography>
+                    ))}
+                </Breadcrumbs>
+                <Stack flexGrow={1} spacing={2} paddingLeft={2}>
+                    {sortedStepsAndContent.map((item) => (
+                        <Box key={item.ID} display="flex" alignItems="center" flexGrow={1}>
+                            {item.type ? (
+                                <StepViewer stepEntity={item} reloadTask={reloadTask} />
+                            ) : (
+                                <TaskContentViewer contentBlock={item} reloadTask={reloadTask} />
+                            )}
+                        </Box>
+                    ))}
                 </Stack>
+                <form noValidate autoComplete="off" onSubmit={handleSubmitNewContent}>
+                    <StyledNewContentBox>
+                        <ViewCompactIcon />
+                        <Tiptap
+                            onContentChange={handleNewContentBlockChange}
+                            entID={task.ID}
+                            initialContent={newContentBlockRef.current}
+                            reloadEditor={reloadEditor}
+                            placeholder={`Add content block to "${task.name}" here...`}
+                            newLineEditor
```

Comment:
Where did the true go here?

Reply:
It behaves similarly right now without the true, if we remove the line entirely then the interface design looks bad, thats it.